include/cvc5/cvc5.h

```diff
@@ -4373,8 +4376,7 @@ class CVC5_EXPORT Solver
    * for showing the satisfiability of the last call to checkSat using the
    * current model. This method will only return false (for any `v`) if
    * option
-   * \verbatim embed:rst:inline :ref:`model-cores <lbl-option-model-cores>`\endverbatim
-   * has been set.
+   * \verbatim embed:rst:inline :ref:`model-cores <lbl-option-model-cores>`\endverbatim has been set.
```

Comment:
This looks longer than 80 char.

Reply:
Looks like I missed reverting the formatting here. I actually didn't intend to change any of the formatting of the text.